src/crypto.js

```diff
@@ -14,15 +14,12 @@
  * limitations under the License.
  */
 
-import {
-  sha384Base64,
-} from '../../third_party/closure-library/sha384-generated';
+import {getElementService} from './element-service';
 
-describe('sha384', () => {
-  it('should hash', () => {
-    expect(sha384Base64('abc')).to.equal(
-       'ywB1P0WjXou1oD1pmsZQBycsMqsO3tFjGotgWkP_W-2AhgcroefMI1i67KE0yCWn');
-    expect(sha384Base64('foobar')).to.equal(
-       'PJww2fZl501RXIQpYNSkUcg6ASX9Pec5LXs3IxrxDHLqWK7fzfiaV2W_kCr5Ps8G');
-  });
-});
+/**
+ * @param {!Window} window
+ * @return {!Promise<!Crypto>}
+ */
+export function cryptoFor(window) {
```

Comment:
Like this https://github.com/ampproject/amphtml/blob/master/build-system/tasks/presubmit-checks.js#L83